frontend/model/contracts/chatroom.js

```diff
@@ -235,17 +235,17 @@ sbp('chelonia/defineContract', {
               !rootGetters.ourContactProfilesById[id]
             )
             sbp('chelonia/contract/sync', profileIds).catch((e) => {
-              console.error('Error while syncing other members\' contracts at chatroom join', e)
+              console.warn('Error while syncing other members\' contracts at chatroom join', e)
             })
           } else {
             if (!rootGetters.ourContactProfilesById[memberID]) {
               sbp('chelonia/contract/sync', memberID).catch((e) => {
-                console.error(`Error while syncing new memberID's contract ${memberID}`, e)
+                console.warn(`Error while syncing new memberID's contract ${memberID}`, e)
               })
             }
           }
         }).catch((e) => {
-          console.error('[gi.contracts/chatroom/join/sideEffect] Error at sideEffect', e?.message || e)
+          console.warn('[gi.contracts/chatroom/join/sideEffect] Error at sideEffect', e?.message || e)
```

Comment:
There are a lot of errors that got converted to warnings that are unrelated to calls to publishEvent in this PR.

Ideally we keep most of the errors as errors, and underfit - in other words, it'd be better to have more errors logged as errors in the console, than missing errors we might want to be aware of.

Ideally this PR only changes a couple of lines related to publishEvent that are regularly logged as errors that happen due to expected circumstances.